crates/topos-tce/src/lib.rs

```diff
@@ -104,52 +128,47 @@ pub async fn run(
     let validator_store = ValidatorStore::open(path.clone(), fullnode_store.clone())
         .expect("Unable to create validator store");
 
-    let grpc_context = GrpcContext::default().with_router(
-        GrpcRouter::new(tonic::transport::Server::builder()).add_service(
-            SynchronizerServiceServer::new(SynchronizerService {
-                validator_store: validator_store.clone(),
-            }),
-        ),
-    );
-
     let certificates_synced = fullnode_store
         .count_certificates_delivered()
         .map_err(|error| format!("Unable to count certificates delivered: {error}"))
-        .unwrap();
+        .expect("Unable to count certificates delivered");
```

Comment:
Instead of `.expect`, we can just use `?`. We repeat the same `error` here. So we either remove `map_err`, or use `?` instead of `expect`.

Same for `139` and `144`.